packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -175,6 +180,9 @@ const initialState: SuiteState = {
         confirmExplanationModalClosed: {},
         explanationBannerClosed: {},
     },
+    tradingTerms: {
+        dontShowAgain: {},
+    },
```

Comment:
does not need to be nested object

```suggestion
    dismissedTradingTerms: {
        buy: true,...
    },
```